### PR TITLE
Restablished remote and local directory with completed katas. Previou…

### PR DESCRIPTION
**Pull request by PerryBaran**
…s directory dissolved due to fatal git errors.

### CODE REVIEW

File: test/kata1.fizzBuzz.test.js [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
-  test("returns Buzz when passed a multiple of 5", () => {});
+  it("returns Buzz when passed a multiple of 5", () => {
+    expect(fizzBuzz(5)).toBe('Buzz');
+  });
 
-  test("returns FizzBuzz when passed a multiple 3 and 5", () => {});
+  it("returns FizzBuzz when passed a multiple 3 and 5", () => {
+    expect(fizzBuzz(15)).toBe('FizzBuzz');
+    expect(fizzBuzz(30)).toBe('FizzBuzz');
+    expect(fizzBuzz(90)).toBe('FizzBuzz');
```
Comment (PerryBaran) [inline test/kata1.fizzBuzz.test.js:15]: Shouldn't mix single and double quotation marks. In this repo the linter has been set up to use double quotation marks so that's what should be used. You can see this in vscode because you get red squiggly lines under strings using single quotation marks. If you hover your mouse over them, it will tell you to replace them with double quotations marks

File: src/kata2.booleanToWord.js [hunk 2]
```diff
@@ -1,3 +1,10 @@
-const booleanToWord = (boolean) => {};
+const booleanToWord = (boolean) => {;
+if (boolean) {
+    return 'yes';
+}
+
+return 'no';
```
Comment (PerryBaran) [inline src/kata2.booleanToWord.js:6]: this can be simplified by using the ternary operator i.e. `return boolean ? "yes" : "no"`

File: src/kata2.booleanToWord.js [hunk 3]
```diff
@@ -1,3 +1,10 @@
-const booleanToWord = (boolean) => {};
+const booleanToWord = (boolean) => {;
```
Comment (PerryBaran) [inline src/kata2.booleanToWord.js:1]: the semi-colon at the end of this line shouldn't be there

File: test/kata3.numberToReversedDigits.test.js [hunk 4]
```diff
@@ -1,5 +1,8 @@
 const { numberToReversedDigits } = require("../src");
 
 describe("numberToReversedDigits", () => {
-  test("returns a reversed array of the number's digits", () => {});
+  test("returns a reversed array of the number's digits", () => {
+    expect((numberToReversedDigits(69))).toEqual([9, 6]);
+    expect((numberToReversedDigits(1234))).toEqual([4, 3, 2, 1]);
```
Comment (PerryBaran) [inline test/kata3.numberToReversedDigits.test.js:6]: you have an extra, unneeded, pair of parenthesis surrounding your function calls in the expect statement. can write `expect(numberToReversedDigits(69))` instead of `expect((numberToReversedDigits(69)))`

File: src/kata3.numberToReversedDigits.js [hunk 5]
```diff
@@ -1,3 +1,9 @@
-const numberToReversedDigits = (number) => {};
+const numberToReversedDigits = (number) => {
+  const numberToString = number.toString();
+  const numberArray = numberToString.split("").reverse();
+  const backToNumber = numberArray.map(num => parseInt(num));
```
Comment (PerryBaran) [inline src/kata3.numberToReversedDigits.js:4]: Some more linting issues here, if you ever have red squiggly lines in your code editor make sure to hover your mouse over to see what the error is and try to fix it. Often times it will have a button `Quick Fix... (Ctrl + .)` at the bottom of the suggestion which you can click to fix the issue

File: test/kata4.humanCatDogYears.test.js [hunk 6]
```diff
@@ -1,3 +1,10 @@
 const { humanCatDogYears } = require("../src");
 
-// Look Ma, no handlebars!!!
+describe("humanCatDogYears", () => {
+  test("returns an array of three numbers in human, cat, and dog years respectively when passed human years", () => {
+    expect(humanCatDogYears(10)).toEqual([10, 56, 64]);
+    expect(humanCatDogYears(2)).toEqual([2, 24, 24]);
+    expect(humanCatDogYears(1)).toEqual([1, 15, 15]);
+    expect(humanCatDogYears(0)).toEqual([0, 0, 0]);
```
Comment (PerryBaran) [inline test/kata4.humanCatDogYears.test.js:8]: Good range of tests here, I like that you made specific tests for the edge cases

File: test/kata5.reachDestination.test.js [hunk 7]
```diff
@@ -1,5 +1,8 @@
 const { reachDestination } = require("../src");
 
 describe("reachDestination", () => {
-  test("returns string with estimated time of arrival", () => {});
+  test("returns string with estimated time of arrival", () => {
+    expect(reachDestination(44, 10)).toBe(`I should be there in ${4.5} hours.`);
+    expect(reachDestination(4, 2)).toBe(`I should be there in ${2} hours.`);
```
Comment (PerryBaran) [inline test/kata5.reachDestination.test.js:6]: don't need to use a template literal in these tests, as the number will be converted to a string anyway. i.e. using `"I should be there in 4.5 hours."` will give the same result. You only need to use a template literal if you don't know what the value is (e.g. a variable you're calling in the string) or if you want to perform some operation in the string

File: src/kata5.reachDestination.js [hunk 8]
```diff
@@ -1,3 +1,6 @@
-const reachDestination = (distance, speed) => {};
-
+const reachDestination = (distance, speed) => {
+  const roundHalf = (num) => Math.round(num * 2) / 2;
+  const estimatedTime = roundHalf(distance / speed);
+  return `I should be there in ${estimatedTime} hours.`;
+};
```
Comment (PerryBaran) [inline src/kata5.reachDestination.js:5]: Bit of a nitpick, but if the `estimatedTime = 1`, you return `"...1 hours"` instead of `"...1 hour"`

File: src/kata4.humanCatDogYears.js [hunk 9]
```diff
@@ -1,3 +1,41 @@
...
+      return firstDogYear;
+    }
+    if (dogNum === 2) {
+      return firstDogYear + secondDogYear;
+    }
+    if (dogNum > 2) {
+      return firstDogYear + secondDogYear + remainingDogYear;
+    }
+    return 0;
+  };
```
Comment (PerryBaran) [inline src/kata4.humanCatDogYears.js:34]: These functions could be refactored to be DRY-er, as the functionality of them is very similar

File: src/kata5.reachDestination.js [hunk 10]
```diff
@@ -1,3 +1,6 @@
-const reachDestination = (distance, speed) => {};
-
+const reachDestination = (distance, speed) => {
+  const roundHalf = (num) => Math.round(num * 2) / 2;
```
Comment (PerryBaran) [inline src/kata5.reachDestination.js:2]: I would refactor this to remove the `roundHalf` function, and instead just call Math.round... after calculating the time, as you're only calling it once e.g.
```
const time = distance / speed
const estimatedTime = Math.round(time * 2) / 2
```

File: test/kata2.booleanToWord.test.js [hunk 11]
```diff
@@ -1,5 +1,11 @@
 const { booleanToWord } = require("../src");
 
 describe("booleanToWord", () => {
-  // how do we create specs again???
+  it ('returns yes when passed true', () => {
```
Comment (PerryBaran) [inline test/kata2.booleanToWord.test.js:4]: remove space between `it` and `(`

**Review by PerryBaran — COMMENTED** [review]
Really good all round, I've left some suggestions throughout

Good Stuff:
- All tests pass
- Testing for edge cases

Improvements:
- Linting issues, if you get red squiggly lines in your code make sure to fix them. Often you can fix these quickly by hovering your mouse over the issue to see the error and then selecting `Quick Fix... (Ctrl + .)`.
- Some functions could be simplified and made DRY-er
